Sources/Common/DIGraph.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+public class DIGraph {
```

Comment:
The code in this class is almost identical to what used to exist in `DICommon`, `DIMessagingPush`, etc before. The logic is the same in this class compared to the classes before.